estate/models/estate_property.py

```diff
@@ -0,0 +1,118 @@
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+from odoo.exceptions import UserError
+from odoo.exceptions import ValidationError
+from odoo.tools import float_compare, float_is_zero
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate Property"
+
+    name = fields.Char(string="Title", required=True)
+    description = fields.Text(string="Description")
+    postcode = fields.Char(string="Postcode")
+    date_availability = fields.Date(
+        default=lambda self: fields.Date.today() + relativedelta(months=3),
+        string="Available From", copy=False,
+    )
+    expected_price = fields.Float(string="Property Expected Price", required=True)
+    selling_price = fields.Float(string="Selling Price", readonly=True, copy=False)
+    bedrooms = fields.Integer(string="Bedrooms", default=2)
+    living_area = fields.Integer(string="Living Area (sqm)")
+    facades = fields.Integer(string="facades")
+    garage = fields.Boolean(string="Garage")
+    garden = fields.Boolean(string="Garden")
+    garden_area = fields.Integer(string="Garden Area (sqm)")
+    garden_orientation = fields.Selection(
+        string="Garden Orientation",
+        selection=[
+            ('north', "North"),
+            ('south', "South"),
+            ('east', "East"),
+            ('west', "West"),
+        ],
+    )
+    state = fields.Selection(
+        string="State",
+        selection=[
+            ('new', "New"),
+            ('received', "Offer Received"),
+            ('accepted', "Offer Accepted"),
+            ('sold', "Sold"),
+            ('cancelled', "Cancelled"),
+        ], required=True, copy=False, default='new',
```

Comment:
This is not a list or a dict, they are arguments of the method Selection, no need to add a comma at the end.
```suggestion
        ], required=True, copy=False, default='new'
```